src/ck3/tables/on_action.rs

```diff
@@ -544,4 +544,7 @@ pub const ON_ACTION_SCOPES: &str = "
 		imprisoner = character
 		reason = flag
 	}
+    funeral_wake_pulse = {
```

Comment:
None of them should be added :) I had started adding them and then stopped, because none are called from the game engine. They're all via other on_actions.
I thought I'd removed funeral_wake_pulse again but apparently not.

Fixed